multiply_ui/server/controller.py

```diff
@@ -4,3 +4,13 @@
 def get_parameters(ctx):
     json_text = pkg_resources.resource_string("multiply_ui", "server/resources/processing-parameters.json")
     return json.loads(json_text)
+
+def get_request(ctx, parameters):
+        time_range = parameters["request"]["timeRange"]
```

Comment:
PEP-8: Indentation shall be 4 spaces.

Hint: You can use PyCharm's & Idea's CTRL+ALT+L before commitig.